websites/P/Pandora/presence.ts

```diff
@@ -86,6 +86,27 @@ presence.on("UpdateData", async () => {
         presenceData.smallImageKey = "pause";
         presenceData.smallImageText = (await strings).pause;
       }
+
+      // Even if we're not playing, we still want the album art if it exists
+      let art = document.querySelector<HTMLSourceElement>(
+        ".Tuner__Audio__TrackDetail__img :first-child :first-child"
+      );
+
+      // If the art does not exist, try again but this time with the now playing info
+      if (!art) {
+        art = document.querySelector<HTMLSourceElement>(
+          ".nowPlayingTopInfo__artContainer__art :first-child :first-child"
+        );
+      }
+
+      // If the art still does not exist, try again but this time with the hero image
+      if (!art)
+        art = document.querySelector<HTMLSourceElement>(".HeroCard__image");
```

Comment:
Use logical assignment operators or nullish coalescing
Also remove the useless comments because we can all read javascript

Reply:
Man, you guys really like your code to be hard to understand. Alright, I'll make the changes, but I disagree with the usage of nullish coalescing/logical assignment, I think it makes far more sense using the multiple if statements.